css/mediaqueries/custom-002.html

```diff
@@ -0,0 +1,63 @@
+<!doctype html>
+<html>
+	<head>
+		<title>Test: custom media queries with cycles are invalid</title>
+		<link rel="author" title="Romain Menke" href="https://github.com/romainmenke">
+		<link rel="help" href="https://www.w3.org/TR/mediaqueries-5/#at-ruledef-custom-media">
```

Comment:
In this test it might be good to also test that cyclic custom media queries become undefined even when they have `or` values that also make them clearly true.  For example, these are undefined per spec:

```css
@custom-media --test1-a screen or (--test1-b);
@custom-media --test1-b screen or (--test1-a);
```

Reply:
Good catch!

`screen or (--test1-b)` isn't valid, but `screen, (--test4-b)` is and I think it also covers what you want to test.